CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Change Log
 
+## [2.0.120230303]
+
+* CLI version update that includes:
+  - Minor bug fixes
+  - Address [issue around named port environment variables not being created](https://github.com/Azure/Bridge-To-Kubernetes/issues/165) 
+  - Address [migrating from netwonsoft to stj](https://github.com/Azure/Bridge-To-Kubernetes/issues/134)
+  - Address [fix to AddhostFileEntry tuple](https://github.com/Azure/Bridge-To-Kubernetes/issues/135)
+  - Address [JSon serialization issue](https://github.com/Azure/Bridge-To-Kubernetes/issues/55)
+  - Address [headless service not getting forwarded all the time](https://github.com/Azure/Bridge-To-Kubernetes/issues/167)
```

Comment:
We keep them open until we release so there is no confusion. If I close them before relase it may make people think things are fixed before they are :)